module1/homework/lengthSort/sort.hpp

```diff
@@ -0,0 +1,7 @@
+#pragma once
+#include <deque>
+#include <forward_list>
+#include <string>
+
+bool compare(const std::string& s1, const std::string& s2);
+std::deque<std::string> lengthSort(const std::forward_list<std::string>& lst);
```

Comment:
The same - no new line

Reply:
new line is here but github is not showing it :(